supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc

```diff
@@ -612,6 +612,48 @@ Use all lower case, unless used in a title or at the beginning of a sentence.
 
 *See also*:
 
+[discrete]
+[[sos-report]]
+==== image:images/caution.png[with caution] sos report (noun, adjective)
+*Description*: In RHEL 8 and later, an `_sos report_` is a collection of files that contain configuration details, system information, and diagnostic data.
+Red{nbsp}Hat Support engineers review an `sos report` when troubleshooting system problems.
+
+Write as shown: two words, monospace font.
+Optionally, append a noun such as _archive_ or _file_, especially when referring to multiple `_sos report_` files.
+(In RHEL 7 and earlier, use the one-word term `_sosreport_` instead of `_sos report_`.)
+
+To generate an `sos report` archive, run the `sos report` command (that is, the `sos` command with the `report` argument).
+
+The `sos` in the command name is an acronym for "son of sysreport", and the official pronunciation is "ess-oh-ess".
+Therefore, if you need to use the indefinite article before " `sos report` ", use _an_ not _a_.
+
+*Use it*: yes
+
+*Incorrect forms*: sosreport, SOSreport, SOSREPORT, SOS report, SOS Report, SOS REPORT
+
+*See also*: xref:sosreport[sosreport]
+
+[discrete]
+[[sosreport]]
+==== image:images/caution.png[with caution] sosreport (noun, adjective)
```

Comment:
The use it is marked as "yes", but this uses the "with caution" icon. Can you update the icon for both of these entries?

Reply:
Changed the "Use it" text to "with caution" to ensure that people use the right term for their RHEL version.